### ISSUE TITLE
Error: File Not Found: create_localisation_tables.php.stub

### ISSUE THREAD

**Issue by rabrowne85** [pwweb/localisation#9]
When running the command `artisan vendor:publish` for the package the following error message is generated:

```bash
Can't locate path: </var/www/html/vendor/pwweb/localisation/src/../database/migrations/create_localisation_tables.php.stub>
```

**Comment by frankpde** [comment]
Please check if fixed in 0.2.1-beta

**Comment by rabrowne85** [comment]
This looks to be resolved.
